research/python/data_integration_example.py

```
@@ -0,0 +1,196 @@
+            futures_price=futures_data.futures_price,
+            spot_price=futures_data.spot_price,
+            time_to_expiry_days=futures_data.time_to_expiry_days,
+            risk_free_rate=provider.get_risk_free_rate(),
+        )
+
+        print(f"  Basis Pressure: {basis_result.pressure:.3f}")
+        print(f"  Fair Basis: {basis_result.fair_basis:.2f}")
+        print(f"  Actual Basis: {basis_result.actual_basis:.2f}")
+        print(f"  Divergence: {basis_result.divergence_pct:.3f}%")
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)